### PR TITLE
Fix broken UX journey caused page redesign

### PR DESCRIPTION
**Pull request by tavy88**
The Skills page has been redesigned and the search form
was removed from the results page causing an issue when
the searched skill has no results, as the user has no way to
search again.

## Screenshots

Screen Shot 2019-08-14 at 15 07 23

https://dfedigital.atlassian.net/browse/GET-365

### CODE REVIEW

File: spec/features/check_your_skills_spec.rb [hunk 1]
```diff
@@ -33,7 +33,17 @@
...
+  end
+
+  scenario 'User can return to the search form when no results are found' do
+    visit(check_your_skills_path)
+    fill_in('search', with: 'Embalmer')
+    find('.search-button').click
+
+    click_on('Search again')
+
+    expect(current_path).to eql(check_your_skills_path)
```
Comment (smoothcontract) [inline spec/features/check_your_skills_spec.rb]: Rubocop is suggesting a change to use `have_current_path` matcher here.

File: app/views/check_your_skills/results.html.erb [hunk 2]
```diff
@@ -12,16 +12,14 @@
...
-    <h3 class="govuk-heading-l"><%= t('.title') %></h3>
-    <%= render "shared/search/results_form", search_path: results_check_your_skills_path, scope: 'check_your_skills.results' %>
     <% if @job_profiles.blank? %>
       <p class="govuk-body-m"><%= render '/shared/search/no_results' %></p>
     <% else %>
       <p class="govuk-body-m" ><%= t('.results', count: @job_profiles.total_count) %></p>
       <p class="govuk-inset-text govuk-!-margin-bottom-2"><%= t('.body') %></p>
       <%= render "results_list", job_profiles: @job_profiles %>
     <% end %>
-    <div class="govuk-grid-column-full govuk-!-padding-top-6 govuk-!-margin-bottom-4 govuk-!-margin-top-6 pagination__section">
```
Comment (cwrw) [inline app/views/check_your_skills/results.html.erb]: discussed in person to add class back

File: app/views/check_your_skills/results.html.erb [hunk 3]
```diff
@@ -12,7 +12,6 @@
 
 <div class="govuk-grid-row govuk-!-padding-top-7">
   <div class="govuk-grid-column-two-thirds" >
-    <h3 class="govuk-heading-xl"><%= t('.title') %></h3>
```
Comment (cwrw) [inline app/views/check_your_skills/results.html.erb]: is this intentional as well?

Comment (tavy88) [reply app/views/check_your_skills/results.html.erb]: Ok, it was a bad merge conflict fix as the condition I added is missing.

File: app/views/check_your_skills/results.html.erb (hunk 2 above)
Comment (tavy88) [reply app/views/check_your_skills/results.html.erb]: Ok, it was a bad merge conflict fix here as well. This should have moved actually inside the else.

**Comment by tavy88** [comment]
@cwrw The things you reported happened due to a previously bad merge. I have updated the code now.

File: app/views/check_your_skills/results.html.erb [hunk 4]
```diff
@@ -12,16 +12,17 @@
...
     <% if @job_profiles.blank? %>
       <p class="govuk-body-m"><%= render '/shared/search/no_results' %></p>
```
Comment (cwrw) [inline app/views/check_your_skills/results.html.erb]: can you remove `<p class="govuk-body-m"></p>` and keep only `<%= render '/shared/search/no_results' %>` as this should be wrapped in it, also it's pushing the section down

**Pull request merged**